### PR TITLE
fix(ui): fix upper jaw inline code string spacing

### LINKED ISSUE #52498: [ui]: upper jaw inline code string spacing
The following Markdown:

```markdown
Create a variable called `card_translation` and assign it a translation table to replace all `-` and ` ` characters with an empty string.
```

Renders as:

image

There is no space between the `\` \`` and the proceeding word `characters`.

There appears to be one in the HTML. So, I assume this is a CSS issue:
image

### PR DESCRIPTION
**Pull request by Mariya-ghafoor**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #52498

The issue was resolved by adding a white-space:pre property in the CSS as the browser by default collapses multiple white space unless there is a ```<pre>``` tag in the HTML.

Result:
Screenshot 2024-07-28 at 4 58 18 PM

### CODE REVIEW

**Review by gikf — COMMENTED** [review]
Hey,

Please revert all changes that are not related to the linked issue.

File: client/src/templates/Challenges/classic/editor.css [hunk 1]
```diff
@@ -55,6 +55,10 @@ textarea.inputarea {
   max-width: unset !important;
 }
 
+code {
+  white-space: pre;
+}
```
Comment (ojeytonwilliams) [inline client/src/templates/Challenges/classic/editor.css:60]: I think all the other problems stem from this selector being too broad. The original issue is with code block inside the challenge description, but this affects all code elements.

If you narrow this selector, it should be possible to revert all the other changes.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Comment by Mariya-ghafoor** [comment]
> Hey,
> 
> Please revert all changes that are not related to the linked issue.

Hi. I was just trying to resolve some failing tests. Seems like they are failing even before I make any changes. I was working on Gitpod. Now I am trying to install the project locally and see if the playwright tests for Chromium still fail.

File: client/src/templates/Challenges/classic/editor.css (hunk 1 above)
Comment (Mariya-ghafoor) [reply client/src/templates/Challenges/classic/editor.css:60]: You are right. Thanks for the pointer.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Thanks for clearing this up, @Mariya-ghafoor! One last thing:

**Review by ojeytonwilliams — APPROVED** [review]
Thanks, @Mariya-ghafoor, this works nicely.

LGTM :+1:

**Comment by Mariya-ghafoor** [comment]
> Thanks, @Mariya-ghafoor, this works nicely.
> 
> LGTM 👍

Thanks for your help

**Comment by ojeytonwilliams** [comment]
You're very welcome! 

Also, for other reviewers, the failing test is nothing to do with this PR.

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍

@Mariya-ghafoor Thank you for the PR and congratulations on your first contribution to this repo 🎉
We look forward to your next.

Happy contributing!

**Pull request merged**